switch/switch_main.ml

```diff
@@ -319,8 +319,14 @@ let main ({ Config.daemonize; path; pidfile } as config) =
         end else raise (Does_not_exist dir)
       | _ ->
         () );
+
     if daemonize
-    then Lwt_daemon.daemonize ();
+    then begin
+      info "Daemonizing...";
+      ignore (Daemon.notify Daemon.State.Ready);
```

Comment:
Looks like we call `notify` immediately after starting. So why use the "notify" type at all, and not just "simple"?